mono/mini/mini-ppc.c

```diff
@@ -734,6 +734,9 @@ void
 mono_arch_flush_icache (guint8 *code, gint size)
 {
 #ifdef MONO_CROSS_COMPILE
```

Comment:
Why is cross compile broken?
Can we not cross compile a JIT?

Reply:
we don't need to invalidate the instruction cache as we don't try to execute it right away

Reply:
Cross compile might not mean what I though. I thought, for some reason, we check if build != host/target, and that is what cross_compile meant. Which wouldn't lead to this.
If it is host != target, then aot only, then understood.
i.e. are we cross compiling mono (first case), or building a cross compiler (second case).

Reply:
it's `host != target`:
https://github.com/mono/mono/blob/4260bd28d0c025ce7e4cbc3734a519d6fabe9279/configure.ac#L4114-L4115